site/docs/configuration.md

```diff
@@ -35,6 +35,20 @@ The properties can be manually constructed or passed in from a compute engine li
 Spark uses its session properties as catalog properties, see more details in the [Spark configuration](#spark-configuration) section.
 Flink passes in catalog properties through `CREATE CATALOG` statement, see more details in the [Flink](../flink/#creating-catalogs-and-using-catalogs) section.
 
+### Lock catalog properties
+
+Here are the catalog properties related to locking. They are used by some catalog implementations to control the locking behavior during commits.
+
+| Property                          | Default            | Description                                            |
+| --------------------------------- | ------------------ | ------------------------------------------------------ |
+| lock.impl                         | null               | a custom implementation of the lock manager, the actual interface depends on the catalog used  |
+| lock.table                        | null               | an optional auxiliary table for locking                |
```

Comment:
Isn't the table required if you're using Dynamo? I would not say it is optional, although it is optional if you're not using Dynamo.

Reply:
Yes, this is not a part of AWS documentation, that is why I say it as optional, I can remove that. I will emphasize it is required in the AWS doc.

Reply:
Thanks for pointing that out. You're right that it is optional here. Thanks!

Reply:
You may also want to link to the Dynamo docs if you haven't already, since that's the only implementation currently available.